src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -105,6 +105,7 @@ public static void ZeroLengthContent_RoundTrip()
                 }
                 catch (CryptographicException) when (PlatformDetection.IsFullFramework) // Expected on full FX
                 {
+                    return;
```

Comment:
I agree with that thinking.  Perhaps we should log a bug for netfx to fix it in the future.  Dan, would you like to do the honors?